plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -318,7 +323,7 @@ def _onAddDevice(self, name, address, properties):
             device = ClusterUM3OutputDevice.ClusterUM3OutputDevice(name, address, properties)
         else:
             device = LegacyUM3OutputDevice.LegacyUM3OutputDevice(name, address, properties)
-
+        self._application.getMachineManager().addDiscoveredPrinter(device.getId(), name, self._createMachineFromDiscoveredPrinter, properties[b"printer_type"].decode("utf-8"))
```

Comment:
The CloudOutputDeviceManager also needs implementation for the add and remove discovered printer functions